### DISCUSSION TITLE
Updating GitLab Documentation on The GitVersion Website

### DISCUSSION THREAD

**Discussion by DarwinJS** [GitTools/GitVersion#2767] [Q&A]
Hello,
I have created a working example of using GitVersion in GitLab. The main trick being running GitVersion and passing along it's version determination throughout the rest of the pipeline.

It is implemented as a an include (dependency) here: https://gitlab.com/guided-explorations/devops-patterns/utterly-automated-versioning/-/blob/develop/gitversion-ci-cd-plugin-extension.gitlab-ci.yml

With an example of using it to do many things here: https://gitlab.com/guided-explorations/devops-patterns/utterly-automated-versioning/-/blob/develop/.gitlab-ci.yml

I thought about making one combined example and amending the documentation here: https://gitversion.net/docs/reference/build-servers/gitlab

But I wanted to validate that this would be the best way to contribute this knowledge back to the documentation?

**Comment by asbjornu** [comment]
Useful additions to the documentation is always welcome! 🙏🏼
